static/builder.html

```diff
@@ -374,6 +374,7 @@
                         <div class="checkboxGroup"><input id="excludeTMR5" type="checkbox"/><label>Exclude TMR of base 5* units</label></div>
                         <div class="checkboxGroup" data-server="JP"><input id="excludeSTMR" type="checkbox"/><label>Exclude STMR</label></div>
                         <div class="checkboxGroup"><input id="excludeNotReleasedYet" type="checkbox" checked="true"><label>Exclude not released yet equipments</label></div>
+                        <div><a class="link" onclick="displayEquippableItemList();">Select items to exclude</a></div>
```

Comment:
Minor : should be displayEquipableItemList, with only one 'p', no ?

Reply:
Good catch!